### PR TITLE
[Internal Manager] Implement internal balances component UI

### PR DESCRIPTION
**Pull request by luizakp**
https://linear.app/bleu-llc/issue/BAL-243/implement-internal-balances-component

This PR add:
- UI following figma
- enable transfer transaction

https://www.loom.com/share/7c8400b60c574c31af33b266a5c7841c

### CODE REVIEW

File: packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
 "use client";
 import { InternalBalanceQuery } from "@balancer-pool-metadata/gql/src/balancer-internal-manager/__generated__/Mainnet";
 import { networkFor } from "@balancer-pool-metadata/shared";
+import {
+  MinusCircledIcon,
+  PlusCircledIcon,
+  WidthIcon,
```
Comment (mendesfabio) [inline packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx:7]: there's not a circled option for this icon, right? could you style it to look like the others?

File: packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx [hunk 2]
```diff
@@ -34,32 +39,29 @@ export function TokenTable() {
...
-            </Table.Body>
-          </Table>
-        )}
-      </div>
+    <div className="flex-1 flex w-full text-white justify-center">
+      {tokensWithBalance && tokensWithBalance?.length > 0 && (
+        <Table>
+          <Table.HeaderRow>
+            <Table.HeaderCell>
+              <span className="sr-only">Token Logo</span>
```
Comment (mendesfabio) [inline packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx:47]: this 1st column (token logo) is a bit large, could you decrease the width a bit?

File: packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx [hunk 3]
```diff
@@ -93,38 +95,113 @@ function TableRow({
...
+  icon: React.ReactElement;
+  operation: string;
+  disabled?: boolean;
+}) {
+  return (
+    <>
+      {!disabled ? (
         <Link
-          href={`/internalmanager/${network}/withdraw/${token.tokenInfo.address}`}
+          href={`/internalmanager/${network}/${operation}/${token.tokenInfo.address}`}
```
Comment (mendesfabio) [inline packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx:180]: we're passing network as props through 3 components to construct this URL, can't we infer it from the route path?

File: packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx [hunk 4]
```diff
@@ -34,32 +39,29 @@ export function TokenTable() {
...
-                  chainId={chain!.id?.toString?.()}
-                  userAddress={addressLower as `0x${string}`}
-                />
-              ))}
-            </Table.Body>
-          </Table>
-        )}
-      </div>
+    <div className="flex-1 flex w-full text-white justify-center">
+      {tokensWithBalance && tokensWithBalance?.length > 0 && (
```
Comment (mendesfabio) [inline packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx:43]: could you add a text for when the user doesn't have an internal balance? e.g. you don't have any internal balance on {{blockchain}}, make a deposit or change de network

**Review by mendesfabio — APPROVED** [review]
looking good!

File: packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx (hunk 3 above)
Comment (luizakp) [reply packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx:180]: the TokenTable component doesn't have the network on the URL yet. But we will do that on[ this issue](https://linear.app/bleu-llc/issue/BAL-310/nested-route-according-to-chain)

**Pull request merged**